dispatcher/stateful.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = function StatefulDispatcher(dispatcher, data, onData) {
```

Comment:
Oh. Copy+Paste mistake. Just wanted to highlight that a short description would help.